<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapMaker</class>
 <widget class="QMainWindow" name="MapMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>524</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MapMaker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/icons/lightsoff.svg</normaloff>:/images/icons/lightsoff.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="addTileset">
        <property name="text">
         <string>agregar</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">addOrQuit</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="removeTileset">
        <property name="text">
         <string>quitar</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">addOrQuit</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="tierra">
        <property name="text">
         <string>tierra</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tilesetToAdd</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bosque">
        <property name="text">
         <string>bosque</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tilesetToAdd</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1080</width>
         <height>1024</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>en este lugar se insertan los partes del mapa</string>
       </property>
       <property name="toolTipDuration">
        <number>5000</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="TerrainPainter" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1080</width>
            <height>1024</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>4096</width>
            <height>4096</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGardarComo"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGardarComo">
   <property name="text">
    <string>Guardar como...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Manual de uso</string>
   </property>
   <property name="toolTip">
    <string extracomment="see the help">Manual de uso</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TerrainPainter</class>
   <extends>QWidget</extends>
   <header>terrainpainter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addTileset</tabstop>
  <tabstop>removeTileset</tabstop>
  <tabstop>tierra</tabstop>
  <tabstop>bosque</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addTileset</sender>
   <signal>clicked()</signal>
   <receiver>MapMaker</receiver>
   <slot>changeMouseAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeTileset</sender>
   <signal>clicked()</signal>
   <receiver>MapMaker</receiver>
   <slot>changeMouseAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tierra</sender>
   <signal>clicked()</signal>
   <receiver>MapMaker</receiver>
   <slot>setActiveTileSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bosque</sender>
   <signal>clicked()</signal>
   <receiver>MapMaker</receiver>
   <slot>setActiveTileSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGardarComo</sender>
   <signal>triggered()</signal>
   <receiver>MapMaker</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbrir</sender>
   <signal>triggered()</signal>
   <receiver>MapMaker</receiver>
   <slot>loadData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGuardar</sender>
   <signal>triggered()</signal>
   <receiver>MapMaker</receiver>
   <slot>simpleSaveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNuevo</sender>
   <signal>triggered()</signal>
   <receiver>MapMaker</receiver>
   <slot>createNewMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeMouseAction()</slot>
  <slot>setActiveTileSet()</slot>
  <slot>saveData()</slot>
  <slot>loadData()</slot>
  <slot>simpleSaveData()</slot>
  <slot>createNewMap()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="addOrQuit">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="tilesetToAdd"/>
 </buttongroups>
</ui>
